### PR TITLE
Update UI.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor fixes. Changed **Food (Distribution) Table** to **Refreshment Table**. Sounds much better in English and shorter. 
Death/Kills are part of PvP, so it's natural to change it to plural form.

Good to merge.

### CODE REVIEW

File: UI.tsv [hunk 1]
```diff
@@ -1691,14 +1691,14 @@ UI_20150714_001690	×%s
...
-UI_20150717_001694	{#ffff00}%s{/} Players like this character
+UI_20150717_001694	{#ffff00}%s{/} players like this character
 UI_20150717_001695	{@st41b}{s56}This game is {#0066FF}
 UI_20150717_001696	Rated T.
 UI_20150717_001697	{s40} {nl}Long hours playing the game can negatively affect your health.{/}
 UI_20150717_001698	{@st66b}Enter
 UI_20150717_001699	{@st42}My Summon Info
 UI_20150717_001700	Your movement slows down when you do not have enough{@st59}STA{/}
-UI_20150717_001701	{@st59}Sort Inventory{/}
+UI_20150717_001701	{@st59}Sorts your inventory{/}
```
Comment (laurencee) [inline UI.tsv:1701]: The original was fine presuming this text applies to a tooltip/button.

Either way it's fine.

File: UI.tsv [hunk 2]
```diff
@@ -1490,7 +1490,7 @@ UI_20150414_001489	{@st44}{s26}Level
 UI_20150414_001490	{@st59b}{s26}ea
 UI_20150414_001491	{@st41b}Cooking
 UI_20150414_001492	{@st41b}Food
-UI_20150414_001493	{@st42}Disband Food Table
+UI_20150414_001493	{@st42}Disband Refreshment Table
```
Comment (Soukyuu) [inline UI.tsv]: Is "disband" the correct word here? You usually disband a group...

File: UI.tsv (hunk 1 above)
Comment (Sunaries) [inline UI.tsv:1701]: It's fine, it adds more flair to it. :smile_cat:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**